packages/webapp/src/stories/Pages/Task/AddSoilAmendmentProducts/AddSoilAmendmentProducts.stories.tsx

```diff
@@ -87,6 +92,11 @@ export const ReadOnly: Story = {
   args: {
     isReadOnly: true,
     farm: { interested: true, country_id: 37, farm_id: 'xxx' },
-    defaultValues: [{ ...defaultValues, product_id: 3 }],
+    defaultValues: [
```

Comment:
Having values for the quantity might be good for the read-only specifically.

Reply:
@Duncan-Brain I don't believe that this story is set up to take default values for the task product (as opposed to product) fields... but I'm basing that mostly off of the purpose not being populated, for instance.

If I'm not mistaken, I don't think _task_ properties population into the `<ProductCard />` is set up yet, is that correct @SayakaOno? Maybe I'll ping on Slack because I'm a bit confused about the final form setup too for readonly.